operator/main.go

```diff
@@ -73,11 +75,15 @@ var (
 	// centralLabelSelector is a kubernetes label selector that is used to filter out Central instances
 	// to be managed by this operator. If the selector is empty, all Central instances are managed.
 	// see https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
-	centralLabelSelector = env.RegisterSetting(keyCentralLabelSelector, env.WithDefault(""))
+	centralLabelSelector = env.RegisterSetting(envCentralLabelSelector, env.WithDefault(""))
 	// securedClusterLabelSelector is a kubernetes label selector that is used to filter out Secured Cluster
 	// instances to be managed by this operator. If the selector is empty, all instances are managed.
 	// see https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
-	securedClusterLabelSelector = env.RegisterSetting(keySecuredClusterLabelSelector, env.WithDefault(""))
+	securedClusterLabelSelector = env.RegisterSetting(envSecuredClusterLabelSelector, env.WithDefault(""))
+	// disableCentralReconciler skips registering central reconciler if set to true
+	disableCentralReconciler = env.RegisterBooleanSetting(envDisableCentralReconciler, false)
+	// disableSecuredClusterReconciler skips registering secured cluster reconciler if set to true
+	disableSecuredClusterReconciler = env.RegisterBooleanSetting(envDisableSecuredClusterReconciler, false)
```

Comment:
nit: change to `envEnableCentralReconciler` and default to `true`, same for SecuedCluster. Reading double negations is harder and leads to misconfigurations.
For example, `ROX_DELEGATED_SCANNING_DISABLED` was used in our debugging yesterday, and by accident we set it to `true` which disabled the feature.